db/test/unit/tables/ccbc_guest_test.sql

```diff
@@ -0,0 +1,41 @@
+begin;
```

Comment:
I believe this should all stay in the same file. This test is in the `/tables` folder which was named after `ccbc_user` table.

It is modelled after this one but ours needs work: 
https://github.com/bcgov/cas-cif/blob/develop/schema/test/unit/tables/cif_user_test.sql

This morning in our meeting Alec asked us to rename ours for consistency here:
https://github.com/bcgov/CONN-CCBC-portal/pull/429/files

Though ours is a work in progress.

Reply:
Ok, lets merge this as is and re-factor tests in another ticket :)